tools/webidl_binder.py

```diff
@@ -59,6 +59,8 @@ def getExtendedAttribute(self, _name):
 parser = argparse.ArgumentParser()
 parser.add_argument('--wasm64', action='store_true', default=False,
                     help='Build for wasm64')
+parser.add_argument('--no-mangled-exports', dest='mangle_exports', action='store_false', default=True,
+                    help='Dont use mangled export names')
```

Comment:
```suggestion
                    help='Avoid use mangled export names')
```
This avoids `Dont` with an apostrophe, which is a typo.